conn.go

```diff
@@ -838,10 +841,6 @@ func (c *Conn) writeClose(p []byte, cerr error) error {
 	return nil
 }
 
-func init() {
```

Comment:
I'd prefer this stays here as it's only used with Ping.

Reply:
ok